src/dispatch/static/dispatch/src/task/Table.vue

```diff
@@ -30,7 +30,7 @@
               :sort-by.sync="sortBy"
               :sort-desc.sync="descending"
             >
-              <template v-slot:item.creator="{ item }">
+              <template v-slot:item.creator.individual_contact.name="{ item }">
```

Comment:
Shouldn't this be `individual` not `individual_contact` based on the Participant model?

Reply:
No, the slot name has to match the value defined as the value in the header value. It has no reflection on the actual item data structure.

Reply:
Got it.